### DISCUSSION TITLE
Where is the serversettings.json file?

### DISCUSSION THREAD

**Discussion by maxtechwell** [jagrosh/MusicBot#1277] [Questions]
I want to give permissions out and don't see it in the config.txt so it must be in this 'serversettings.json' file. The file isn't in the folder, and I didn't see it mentioned in the setup guide. What am I doing wrong here?

`[16:04:21] [INFO] [JMusicBot]: Loaded config from \\\`
`[16:04:21] [WARN] [Settings]: Failed to load server settings (this is normal if no settings have been set yet): `
`java.nio.file.NoSuchFileException: serversettings.json`
`[16:04:22] [INFO] [JDA]: Login Successful!`
`[16:04:22] [INFO] [WebSocketClient]: Connected to WebSocket`
`[16:04:22] [INFO] [JDA]: Finished Loading!`

**Comment by jagrosh** [comment]
You're not doing anything wrong; it's normal to see that message if the file doesn't exist. The file only gets generated if you set some settings on a server. Also note that you should not be manually editing that file, so you can completely ignore that warning unless you're seeing some other issue with server settings not persisting past a restart.
